README.md

```diff
@@ -126,75 +145,91 @@ Or the entire project (which will take a very long time!):
+Or, you can selectively run the generator on one or more services:
 
 ```bash
-bazel build //lib/...
+./scripts/generate ec2 s3 iam
 ```
 
-To view what targets are available in the workspace:
+To update the [botocore](botocore) service definitions used by the generator, you can run:
 
 ```bash
-bazel query //...
+./scripts/update-botocore
 ```
 
-> By default, the `bazel` command will use the same GHC version as the Nix shell's `ghcVersion` argument. You can choose a different GHC version using `nix-shell --argstr ghcVersion 884` - which is just a synonym for `bazel build --//tools/ghc:version=884`.
-
-## Building the Documentation
+[scripts/generate-configs](scripts/generate-configs) will run the config generator to produce placeholder [config/services](config/services) configurations for any [botocore] services.
 
-The [docs](docs) Bazel package contains the Haddock target and Hugo static site definition and markdown content. To build the site locally, run:
+To generate configurations for any new/missing services:
 
 ```bash
-bazel build //docs:bundle
+./scripts/generate-configs
 ```
 
-Alternatively, you can serve the documentation site locally on `http://localhost:1313` by running:
+Service configurations generated in this way are intended as examples only and the resulting `configs/services/<name>.json:libraryName` (Haskell package name) and `configs/annexes/<name>.json:serviceAbbreviation` (Haskell package namespace) should be manually verified and curated as necessary.
 
-```bash
-bazel run //docs:serve
-```
+For pull requests which affect generated output please _do not include_ the regenerated `amazonka-*` packages, only commit updates to the build rules, documentation, generator, and related configuration. This is to make code review more manageable by focusing pertinent changes such as configuration and logic changes in pull requests and designates the maintainers and Continuous Integration as the source of truth for the generated code.
 
-## Running the Code Generator
+## Amazonka as a Git Dependency
 
-The [gen](gen) Bazel package contains code generators for synthesising Haskell data types, packages, and configuration from the botocore service definitions.
+If there are as-yet-unreleased features or fixes that have yet to make it to Hackage, you can use the `main` (or another) development branch by declaring Amazonka as a Git dependency by following the Cabal or Stack instructions below.
 
-[scripts/generate](scripts/generate) will run the code generator for all services configured in [config/services](config/services), for example:
+> Note: `amazonka-core` is a required dependency of the main `amazonka` package, in addition to `amazonka-sts` and `amazonka-sso` due to `sts:AssumeRoleWithWebIdentity` and SSO authentication and authorization support. These required dependencies can then be supplemented by any additional service libraries you use from `lib/services/amazonka-<service>`.
```

Comment:
AWS SSO still has that internal name, but maybe we say "SSO via AWS IAM Identity Center"?